package.json

```diff
@@ -2,7 +2,9 @@
   "name": "@lokalise/your-project-name",
   "version": "1.0.0",
   "scripts": {
-    "build": "vite build"
+    "build": "vite build",
+    "lint": "eslint src/.",
+    "lint:fix": "npm run lint -- --fix"
```

Comment:
(This is just personal preference)
I like separation of `lint` and `format`. Then each can be divided further if needed. Also shorter to write.

Reply:
To me, adjusting imports is also formatting. But, again, this was just preference. I'm fine if we leave as is.